### DISCUSSION TITLE
Page deployment stopped working (404).

### DISCUSSION THREAD

**Discussion by VladiStep** [community/community#44824] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've created one static webpage some time ago, and its latest deployment worked as expected.
Then, I made it private and then public again, and as I remember it still worked after that.

And now, it doesn't work - [https://vladistep.github.io/Rainbow_trail_effect/](https://vladistep.github.io/Rainbow_trail_effect/) returns error 404, and I don't know why.

**Comment by MihneaMocanu** [comment]
The link may not be the correct one? Try using https://github.com/VladiStep/Rainbow_trail_effect

**Reply by VladiStep** [reply]
No, I mean the deployment page, not the repository.
The link I mentioned was leading to working custom webpage, and now it returns 404.
